### ISSUE TITLE
Installation fails when WITH_TLS=no

### ISSUE THREAD

**Issue by mattst88** [eclipse/mosquitto#2281]
Reported in the Gentoo bugzilla: https://bugs.gentoo.org/806244

Installation fails when `WITH_TLS` is set to `no` because `mosquitto_passwd` is not built, but is unconditionally installed.

`apps/mosquitto_passwd/Makefile` contains:

````Makefile
ifeq ($(WITH_TLS),yes)
all: mosquitto_passwd
else
all:
endif
[...]
install : all
	$(INSTALL) -d "${DESTDIR}$(prefix)/bin"
	$(INSTALL) ${STRIP_OPTS} mosquitto_passwd "${DESTDIR}${prefix}/bin/mosquitto_passwd"

uninstall :
	-rm -f "${DESTDIR}${prefix}/bin/mosquitto_passwd"
````

**Comment by ralight** [comment]
Thanks @mattst88, that should be fixed  in the `fixes` branch now, and will be part of 2.0.12 soon.

**Comment by mattst88** [comment]
Thank you very much, and sorry for not providing a patch myself!
